### PR TITLE
Bugfix/devsu 1450 fix msi scatter

### PR DESCRIPTION
**Pull request by Anna-log7**
This is to fix the sizing of the 1000x1000 msi scatter image. I added props to the `Image` component to allow it to be set to a custom size which fixes this issue.

### CODE REVIEW

File: app/components/AuthenticatedRoute/index.jsx [hunk 1]
```diff
@@ -48,14 +48,15 @@ const AuthenticatedRoute = ({
...
+        render={(props) => (<ChildComponent admin={admin} {...props} />)}
       />
     </>
   );
 };
 
 AuthenticatedRoute.propTypes = {
-  component: PropTypes.node.isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
+  component: PropTypes.object.isRequired,
```
Comment (kttkjl) [inline app/components/AuthenticatedRoute/index.jsx:59]: would `PropTypes.element` work better?

File: app/components/AuthenticatedRoute/index.jsx [hunk 2]
```diff
@@ -48,14 +48,15 @@ const AuthenticatedRoute = ({
...
   // eslint-disable-next-line react/forbid-prop-types
   location: PropTypes.object,
```
Comment (kttkjl) [inline app/components/AuthenticatedRoute/index.jsx:61]: I thought we had defined a location propType before

Comment (Anna-log7) [reply app/components/AuthenticatedRoute/index.jsx:61]: Not that I see, no

File: app/components/AuthenticatedRoute/index.jsx (hunk 1 above)
Comment (Anna-log7) [reply app/components/AuthenticatedRoute/index.jsx:59]: It complains with element too

File: app/components/AuthenticatedRoute/index.jsx (hunk 2 above)
Comment (kttkjl) [reply app/components/AuthenticatedRoute/index.jsx:61]: Ohh, graphKB then ... out of context for this PR, but 

```
const LocationPropType = PropTypes.shape({
  hash: PropTypes.string,
  key: PropTypes.string,
  pathname: PropTypes.string,
  search: PropTypes.string,
});
```
is what's defined in graphKB

**Review by kttkjl — APPROVED** [review]

**Pull request merged**